### ISSUE TITLE
Can only cat 26 files

### ISSUE THREAD

**Issue by trappist** [tcocca/active_pdftk#35]
If more than 26 files are concatenated, the files labelled AA, AB, AC and so on in the generated command are not readable, resulting in errors:

Error: Failed to open PDF file: 
   AA=/path/to/my1.pdf
Error: Failed to open PDF file: 
   AB=/path/to/my2.pdf

**Comment by tcocca** [comment]
Can you print out the full command that is generated by any chance?  I'm not sure why the files wouldn't be readable. Thanks.

**Comment by mumualex** [comment]
same problem...

```
 Error: Failed to open PDF file: 
  AA=/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-o40rwj.pdf
Error: Failed to open PDF file: 
  AB=/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-l9yv29.pdf
Error: Failed to open PDF file: 
  AC=/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-1ljec7f.pdf
Error: Failed to open PDF file: 
  AD=/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-191lsrt.pdf
Errors encountered.  No output created.
Done.  Input errors, so no output created.
#!# While executing #=> `/usr/bin/pdftk A="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-n8zrxl.pdf" B="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-1sk1s8h.pdf" C="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-1xa26rh.pdf" D="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-yurrfk.pdf" E="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-1d6t3su.pdf" F="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-1sezlz5.pdf" G="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-5qicas.pdf" H="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-1e9udsa.pdf" I="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-qju4wz.pdf" J="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-15w7vod.pdf" K="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-156s7sv.pdf" L="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-1hyh74g.pdf" M="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-1i4xxkb.pdf" N="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-tzhals.pdf" O="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-do1axp.pdf" P="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-ei5hpq.pdf" Q="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-1lbffzi.pdf" R="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-1gus4ye.pdf" S="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-g5vk71.pdf" T="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-xp97sl.pdf" U="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-n4an4j.pdf" V="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-1lax216.pdf" W="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-9wckz6.pdf" X="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-68j197.pdf" Y="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-pf1clg.pdf" Z="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-17mtssd.pdf" AA="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-o40rwj.pdf" AB="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-l9yv29.pdf" AC="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-1ljec7f.pdf" AD="/data/app/apps/myapp/releases/20140514080011/tmp/generated_pdf20140514-9520-191lsrt.pdf" cat A B C D E F G H I J K L M N O P Q R S T U V W X Y Z AA AB AC AD output -`
 /data/app/apps/myapp/shared/bundle/ruby/2.1.0/bundler/gems/active_pdftk-a1065e8f0d21/lib/active_pdftk/call.rb:196:in `block in pdftk'
```

**Comment by tcocca** [comment]
Is this just a limitation of the pdftk library?  Have you tried just passing more than 26 files to that library in the command line itself and taking active_pdftk out of the mix?

**Comment by mumualex** [comment]
sorry, it's the pdftk limitation with version 1.44, it's fixed in PDF Chain 0.4.0 but only supported with pdftk 1.45 and newer, not released in stable debian package

**Comment by tcocca** [comment]
OK, so since this is a limitation of the actual pdftk CLI, I am going to close the ticket.  Thanks for verifying.
